test/usecases/solve/test_finite_difference.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+from neuron import h, gui
+from neuron.units import ms
+
+
+def test_finite_difference():
+    nseg = 1
+
+    s = h.Section()
+    s.insert("finite_difference")
+    s.nseg = nseg
+
+    x_hoc = h.Vector().record(getattr(s(0.5), f"_ref_x_finite_difference"))
```

Comment:
```suggestion
    x_hoc = h.Vector().record(s(0.5)._ref_x_finite_difference)
```

Either this, or just remove the `f` since no value is being replaced in the string.